paddle/operators/math/vol2col.cc

```diff
@@ -48,6 +49,28 @@ class Vol2ColFunctor<platform::CPUPlace, T> {
     int channels_col =
         input_channels * filter_depth * filter_height * filter_width;
 
+    PADDLE_ENFORCE_EQ((input_depth + 2 * padding_depth -
+                       ((dilation_d * (filter_depth - 1) + 1))) /
+                              stride_depth +
+                          1,
+                      output_depth,
+                      "input_depth and output_depth are "
+                      "Mismatching.");
+    PADDLE_ENFORCE_EQ((input_height + 2 * padding_height -
+                       ((dilation_h * (filter_height - 1) + 1))) /
+                              stride_height +
+                          1,
+                      output_height,
+                      "input_height and output_height are "
+                      "Mismatching.");
+    PADDLE_ENFORCE_EQ((input_width + 2 * padding_width -
+                       ((dilation_w * (filter_width - 1) + 1))) /
+                              stride_width +
+                          1,
+                      output_width,
+                      "input_width and output_width are "
+                      "Mismatching.");
```

Comment:
Same as above, whether it needs to check again?

Reply:
I think we can write in this way first, and discuss it later. Because other functors also have similar problem.